prow/tide/tide.go

```diff
@@ -1544,7 +1557,14 @@ func sortPools(pools []Pool) {
 	})
 
 	sortPRs := func(prs []PullRequest) {
-		sort.Slice(prs, func(i, j int) bool { return int(prs[i].Number) < int(prs[j].Number) })
+		sort.Slice(prs, func(i, j int) bool {
```

Comment:
what happens to the merge queue if labels change in a way that modifies priority while a batch job is running? hoping that doesn't interrupt an already-running batch or merge job